pytree/templates/home.html

```diff
@@ -12,7 +12,7 @@ <h1>pytree server</h1>
         Well done, it looks like your wsgi flask pytree server is installed correctly.
         </p>
         <p>
-        <a href="{{ url_for('get_profile') }}?minLOD=0&maxLOD=5&width=10&coordinates={2551900,%201203475},{2555100,%201206475}&pointCloud=sitn2016&attributes=%27" target=_blank>This example</a> request should return a json header followed by a (binary) array buffer to use in GeoMapFish 2.3 using following GET parameters:
+        <a href="{{ url_for('get_profile') }}?minLOD=0&maxLOD=5&width=10&coordinates={2538598,%1180300},{2539573,%1181290}&pointCloud=sit&attributes=%27" target=_blank>This example</a> request should return a json header followed by a (binary) array buffer to use in GeoMapFish 2.3 using following GET parameters:
```

Comment:
Please revert this

Reply:
https://www.youtube.com/watch?v=_Qg3Rk-B09o